app/views/questions/show.html.erb

```diff
@@ -9,15 +8,16 @@
 <% end %>
 
 <h5>Add comment:</h5>
-<%= form_for [@question, @comment] do |f|%>
-  <%= f.label :comment %>
-  <%= f.text_field :text %><br><br>
-  <%= f.submit 'Comment!' %>
-<% end %>
+  <%= render partial: "comments/question_comment", locals: {question: @question, comment: @comment} %>
```

Comment:
This would change to this:
<%= render partial: "comments/question_comment", locals: {target: @question, comment: @comment} %>